src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color ,double width,double height) {
```

Comment:
There is a missing space after the comma in the constructor parameter list `String color ,double width,double height`. It should be `String color, double width, double height`.